database/seeders/SaleSeeder.php

```
@@ -21,46 +21,124 @@ public function run(): void
+                        // Para totales muy grandes, redondear al siguiente múltiplo de 50000
+                        $billete = ceil($totalRedondeado / 50000) * 50000;
+                    }
+                    
+                    $amountPaid = $billete;
+                    $changeAmount = $amountPaid - $total;
+                } else {
+                    // Caso 3: Pago pendiente (5% de probabilidad)
+                    // El cliente paga entre 80% y 95% del total
+                    $porcentajePago = rand(80, 95) / 100;
+                    $amountPaid = round($total * $porcentajePago, -3); // Redondeamos a miles
+                    $pendingAmount = $total - $amountPaid;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pendingAmount'. (Unused Code Rules)

(UnusedLocalVariable)